src/node_env_var.cc

```diff
@@ -72,8 +72,9 @@ static void EnvSetter(Local<Name> property,
                       Local<Value> value,
                       const PropertyCallbackInfo<Value>& info) {
   Environment* env = Environment::GetCurrent(info);
-  if (env->options()->pending_deprecation && env->EmitProcessEnvWarning() &&
-      !value->IsString() && !value->IsNumber() && !value->IsBoolean()) {
+  if (env->options()->pending_deprecation && !value->IsString() &&
```

Comment:
A comment indicating `EmitProcessEnvWarning` should be called at the end would help future readers and refactorings.